src/sync/pull-request.ts

```diff
@@ -181,3 +194,21 @@ export const doGetPullRequestTask = async (
 				: undefined
 	};
 };
+
+const filterEdgesSinceFromDate = (edges: Octokit.PullsListResponseItem[], fromDate: Date | undefined, logger: Logger) => {
+
+	//if there's no from date, just return
+	if (!fromDate) return edges;
+
+	return edges.filter(edge => {
+		try {
+			const edgeCreatedAt = new Date(edge.created_at);
+			//only return edges/pull_requests that are created later than the fromDate in arg
+			return edgeCreatedAt.getTime() > fromDate.getTime();
+		} catch (e) {
+			logger.warn({ fromDate, edgeCreatedAt: edge.created_at }, `Failed to parse created_at from pull requests edges`);
```

Comment:
Is it even possible? Could you please add TODO to remove this catch once we confirm (from splunk?) that this is not possible?